server/infinispan/src/test/java/org/hawkular/apm/server/infinispan/InfinispanSpanCacheTest.java

```diff
@@ -114,4 +111,12 @@ private Span getOne(SpanCache spanCache, String id) throws CacheException {
 
         return Collections.unmodifiableList(Arrays.asList(csAnnotation, crAnnotation));
     }
+
+    private InfinispanSpanCache createCache() {
+        InfinispanSpanCache spanCache = new InfinispanSpanCache();
+        spanCache.setSpansCache(cacheManager.getCache(InfinispanCommunicationDetailsCache.CACHE_NAME +
+                UUID.randomUUID()));
+
```

Comment:
Same comment as above - constructor instead of setter - but just maintain a default constructor.